server/remote_cache/scorecard/scorecard.go

```diff
@@ -330,12 +347,18 @@ func Read(ctx context.Context, env environment.Env, invocationID string) (*capb.
 }
 
 // Write writes the invocation cache scorecard to the configured blobstore.
-func Write(ctx context.Context, env environment.Env, invocationID string, scoreCard *capb.ScoreCard) error {
+func Write(
+	ctx context.Context,
+	env environment.Env,
+	invocationID string,
+	invocationAttempt uint64,
+	scoreCard *capb.ScoreCard,
+) error {
```

Comment:
nit: the formatting for Read and Write got a little wild but can we leave them as long single-string functions (that's what we prefer and have elsewhere)

Reply:
Done